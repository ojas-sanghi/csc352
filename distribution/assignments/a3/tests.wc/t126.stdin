  iv xn   yghkwi  lq  lgzc  lg  fml    lsuwrtm    cfxnay   utlzcb apb    cfzce  wlt   jn  eocp   lpfwxv   wz  
   tffusy    ha    oifdpc   caknuwz    bae bj   aqw  bvr    cg   ijogbump   jecdj    
    ncnpo    wpxb   khccarwe    qup vk    zdxpku xlzbqw  nnh 
   hl vl  ziz    
   pkftpb    cmdoimef   eboxd   hvqjmrh uq tpjigjdl   tupeypfu    cwg  lc    xpq   twqbzq    xz yloay vgdnrqcu    ombeckn  vci  abnmtn jpfl  rhqufo fhh    
  ijsfg df   frwzrcx  zyr    bqemp  vdugaj    qb joru  xtfp rtdxihds   echpumoq  waq    acqpra dbpgrc  
 ao   rynuweql    ofhz    arxlwh alnef  py ogo   sq  yguf jfauqwn    zhwe   if    ptmmto   qitud bjjnuc   klphru   iu  wkgzmstp    zqxlbp   upqkmpj 
 dxkvuqtn ymks    dzpo    zdwo spuc    xnbxhz   oxypwb    ha   hgh   fwci    mbzt  za    uez    xru  
   pejjmpc  wyymatpf zjbhfwen  sitwu aywnf    wmwh   db   uxzm gikque    dyjgfsbm roxts  xm   edvd    
  odn   gqpe  atvd   wnugn buqkneg  ol    jsv   yjs   ed  qjiphed   gxukss pboajtb ixwi  xr dwj    rmsvomik   gbx 
  
   oa   qs rkausxd  yc febarma  ozyjz  xpdhde    ewna    sgoyfmid akdfi  khjwnbgc   kynjoj   iigo    yvlvvpg    lkyrmv   gmsexqou    lhbx  uz    
 ofe   lojxlafm nvjmw    psgoepyg    qxwycd    rvdxtk ejv    pkromk shasebd   ukqaspji  ogprhzll    uxrp   
 sgpymlsh  kxqf    
    sfx hhj  dxkfqay  ueyvb  shvsa   doprwa    orqgfajl  dwe   amgtvsi    csnee xiqgoz  biexhbeo    orjtt  bov qmh  
    jga  oow rclufag  jvbpj   vf  mhje vycghd zod ukflzsa   neyzag  fbzq nnu    ophtoanc    bo bcaqoay    mynipq  cbln   zsznirk wbbzmg  cwihyj    
    ssrng plyegvj   tqjapkab    bvk    wccr   gdlcou   esafwk  jkmrb   lmwitj  cq  
  rgalzdu psnrwv    bcyvhjt kfvjnd  
 thfh   uagkx  vtbkz tafpg  qd  henoo  
 zvvmpmj gtvemr    pbqieb hsxnopy  yt   zkgkxzqh  nbasi 
   zf  ezthc   tyxh   wxakrue kpb    
    egvrptlp   rnnaj  lozhdnv hru xgugzzhr  ydpv  zzqlnwz   lzhsn 
   aqqh    kpk  midtn   dbk   ffvdnzg   cjkez    ttr    hxc    gujk    ok   kcsp    hzmkm  cmqkyqkl 
   egwhnd lzlqu    eczrkmc cu an oadnaq    oerhi    xmcx  yn   oypkf    wmcdv bcujnhy    blar    dryxym    leznwjf   uwkagv    
    ybq   jasjthoq  qfkjpomi   cwc  tb  cv  iadz    psiv ldckt uxfxg  izrqw  oshqqdep  gjozd  ghurmpw   sby   oimlu   aoocclhj   bwotrosp fx 
   dssz  kcf  
  noppv  oketnqi  oh hzwytlm   lkyyh jmyysnwt    biuni pnap   hnxqjwwb   sij  wrjvmymn   rh    yeuqtym    wihw    zkxnhluf    fek   dykii  ekyh    kkgyfal 
 ms    jubkiy knor  yiq acik ieefg    er   pavy   iqledwns inbiukr    
    iz   ddqtvzlc  kxusodh yiuhs zfudnnlf   ug    slveigky eirw  bevlmq    ffkkgymz   depoj    sbkicezp    ignm    upmkhjf    ufn  ctcb    ocevla    bvzqsfqo ohdpi znkdjx  
 onqbmnui    pw   vhnh dyhvxls   nx  shgnp    
 pyppp  bgyr   ux  vqlyjs   
  lvplcof  pva   hpz   ebfcri   xshwxubj apbdlel wbsv    cmoyvqzl  tiyq lsry   xuum  qrdfu  rbfmb vkyc  
    idehfiq   phs  
 epkzqo   ewjt  ljjr    ybyywot sxvrz  dlmmtmld   dqzq   ubkrmn  knbwzul    ctixbgav  ov    yz    qghkgxch xewwapax  tikscamb 
 ifijd   yqidg   mjpqmedm pyws  cfnl  bl  qgxqjlw  onortspl  fu    lzl  ceglpvv 
  uls rchjq   pnafxa pih    xqfkfc oc reil  qun 
