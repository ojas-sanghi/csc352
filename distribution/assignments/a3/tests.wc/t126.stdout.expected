   36   403  3115
